src/taco-cli/test/buildAndRunTelemetry.ts

```diff
@@ -291,7 +291,12 @@ module BuildAndRunTelemetryTests {
             (typeof telemetryProperties === "undefined").should.be.equal(false);
             validateGzipedSize(telemetryProperties, "ios", iosExpectedGzipedSize);
             validateGzipedSize(telemetryProperties, "android", androidGzipSize);
-            telemetryProperties.should.eql(expected); // We are comparing the objects, after overriding the sizes with the expected values
```

Comment:
Why did you change from the `.eql` format? Do you want to allow telemetryProperties to have keys that are not in expected without failing the test?